.env.sample

```
@@ -32,6 +32,10 @@ L1_ENDPOINT_WS=
 # HTTP RPC endpoint of a L1 beacon node.
 L1_BEACON_HTTP=
 
+# REQUIRED FOR PROVER
+# If you are running your Taiko node on a separate server from your sgx prover, please provide the server hosting the taiko node's public IP address here (no slash at the end). Otherwise, if you are running both on one machine, leave it as is.
+TAIKO_NODE_IP=http://l2_execution_engine
```

```suggestion
TAIKO_NODE_IP=
```
Lets keep it empty when its required?